### ISSUE TITLE
improve initial state

### ISSUE THREAD

**Issue by serusko** [romainberger/react-portal-tooltip#95]
I have one small improvement proposal of StatefulTooltip.

from nature of react, there is one case when this tooltip does not work as expected
imagine when u are using StatefulTooltip, 
and component is created at moment, when mouse is already in place where tooltip **will be** rendered, but there is no mouse event fired, so we cannot see tooltip

there is easy way how to reproduce this problem:
- go over your "tooltiped" element
-> tooltip is displayed, OK
- press shortcut for refreshing page (cmd+r / f5 ...)
-> tooltip its not displayed

proposed solution: 

calculate collision of Element.getBoundingClientRect and pointer position on didMount

**Comment by durasj** [comment]
Agreed, I have this same problem. Do you please need any more info @romainberger?
